templates/web/base/admin/extra-metadata-form.html

```diff
@@ -75,6 +81,10 @@
                             [% loc('Name') %]
                             <input class="js-metadata-option-name" name="metadata[[% outer_loop.index %]].values[[% loop.index %]].name" type="text" value="[% option.name | html %]">
                         </label>
+                        <label>
+                            [% loc('Disable form') %]
+                            <input class="js-metadata-option-disable" name="metadata[[% outer_loop.index %]].values[[% loop.index %]].disable" type="checkbox"[% ' checked' IF option.disable %]>
+                        </label>
```

Comment:
This needs to be added to the `renumber_metadata_options` method in `admin.js` otherwise if you check it when you add the option for the first time it doesn't get set.